user_tools/src/spark_rapids_tools/cmdli/argprocessor.py

```diff
@@ -632,13 +716,20 @@ class PredictUserArgModel(AbsToolUserArgModel):
     """
     qual_output: str = None
     prof_output: str = None
+    estimation_model_args: Optional[Dict] = dataclasses.field(default_factory=dict)
 
     def build_tools_args(self) -> dict:
+        if self.estimation_model_args is None or not self.estimation_model_args:
+            def_model = QualEstimationModel.XGBOOST
+            self.p_args['toolArgs']['estimationModelArgs'] = QualEstimationModel.create_default_model_args(def_model)
```

Comment:
nit: Can we use `QualEstimationModel.get_default()` here?

Reply:
I find this block repeated multiple times where we always pass the `def_model` to `create_default_model_args()`. 

Can we directly initialise the dict inside `create_default_model_args()` as:

```python
return {
            'estimationModel': cls.get_default(),
...
```

Reply:
I thought about that.
The reason I did not go for it is that `cls.get_default()` might change its return. but in some call sites, we want to make sure it calls a specific ENUM value.

Reply:
Okay. In that case we should good then.